src/main/java/org/certificatetransparency/ctlog/comm/HttpLogClient.java

```diff
@@ -11,11 +11,7 @@
 import org.apache.commons.codec.binary.Base64;
 import org.apache.http.NameValuePair;
 import org.apache.http.message.BasicNameValuePair;
-import org.certificatetransparency.ctlog.CertificateInfo;
-import org.certificatetransparency.ctlog.CertificateTransparencyException;
-import org.certificatetransparency.ctlog.ParsedLogEntry;
-import org.certificatetransparency.ctlog.ParsedLogEntryWithProof;
-import org.certificatetransparency.ctlog.SignedTreeHead;
+import org.certificatetransparency.ctlog.*;
```

Comment:
Please revert this as Google Java style does not use wildcards. See https://google.github.io/styleguide/javaguide.html/